<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FEDEClass</class>
 <widget class="QMainWindow" name="FEDEClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FEDE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>691</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>评估选择</string>
     </attribute>
     <widget class="QFrame" name="frame_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>471</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>340</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>水域搜救场景</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>140</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>工业事故场景</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>建筑火灾场景</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>340</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>地铁与机场场景</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>140</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>建筑坍塌事故场景</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>340</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>公路交通事故场景</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_54">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/FEDE/Resources/fire_black.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_55">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>81</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/FEDE/Resources/building_black.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_56">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>73</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/FEDE/Resources/factory_black.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_57">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>270</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/FEDE/Resources/road_black.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_58">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>270</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/FEDE/Resources/traffic_black.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_59">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>270</y>
         <width>81</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/FEDE/Resources/logo_black.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_frame_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>651</width>
         <height>451</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_frame_7</zorder>
      <zorder>pushButton_6</zorder>
      <zorder>pushButton_3</zorder>
      <zorder>pushButton</zorder>
      <zorder>pushButton_5</zorder>
      <zorder>pushButton_2</zorder>
      <zorder>pushButton_4</zorder>
      <zorder>label_54</zorder>
      <zorder>label_55</zorder>
      <zorder>label_56</zorder>
      <zorder>label_57</zorder>
      <zorder>label_58</zorder>
      <zorder>label_59</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>评估记录</string>
     </attribute>
     <widget class="QLabel" name="label_frame_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>641</width>
        <height>451</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>601</width>
        <height>411</height>
       </rect>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>附加设置</string>
     </attribute>
     <widget class="QLabel" name="label_frame_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>641</width>
        <height>451</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>371</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_color">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_77">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>色彩设置</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_size">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_78">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>标题大小</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_color">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>1101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>事故应急演练评估平台</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(65, 130, 195);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>54</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/FEDE/Resources/all.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>83</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currecttime_1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>83</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>label_47</zorder>
   <zorder>label_12</zorder>
   <zorder>label_logo</zorder>
   <zorder>label_14</zorder>
   <zorder>label_currecttime_1</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fede.qrc"/>
 </resources>
 <connections/>
</ui>
